isthmus/src/test/java/io/substrait/isthmus/KeyConstraintsTest.java

```diff
@@ -12,11 +13,8 @@ public class KeyConstraintsTest extends PlanTestBase {
   public void tpcds(int query) throws Exception {
     SqlToSubstrait s = new SqlToSubstrait();
     String[] values = asString("keyconstraints_schema.sql").split(";");
```

Comment:
since the `public static List<SubstraitTable> processCreateStatements(String createStatements)` method seems to be able to parse multiple CREATE statements from a single string I'm wondering whether we really need to split the schema SQL file content anymore. I do understand we are missing a `public static CatalogReader processCreateStatements(String createStatements)` method to properly replace this.